### ISSUE TITLE
Default targets for preset-env shall be “defaults”

### ISSUE THREAD

**Issue by dilyanpalauzov** [babel/babel#11546]
No `.browserslist` file implies a `.browserslist` file with content `default`, and this means support for IE 11 and this means, as of https://kangax.github.io/compat-table/es6/, only some ES2015/ES6.

https://babeljs.io/docs/en/babel-preset-env#targets says “Sidenote, if no targets are specified, @babel/preset-env will transform all ECMAScript 2015+ code by default.”

Apart from the fact that there is a “caveat” dead-link-hint above this citation, assuming the wording means ”will transform TO ES2015+ by default" (thus it will run on any ES2015 javascript engine), please change the default of no "targets", do mean `targets: "defaults"`.

**Comment by existentialism** [comment]
@dilyanpalauzov the initial intent of `preset-env` was to replace `preset-latest`, that is, transform all ES2015+ code down to ES5.

Changing this behavior is a breaking change. Check out a few relevant discussions: https://github.com/babel/babel/pull/8897#issuecomment-490659840, https://github.com/babel/babel/issues/9962, and ultimately the PR that will land in Babel 8: https://github.com/babel/babel/pull/10897

**Comment by dilyanpalauzov** [comment]
The default for preset-latest means es2015, es2016, and es2017.  But the default for preset-env is “Sidenote, if no targets are specified, @babel/preset-env will transform all ECMAScript 2015+ code by default.” 

This needs rewording like, “the resulting code will require ES2017 conformant engine”.  I really do not understand what is written there.

**Comment by existentialism** [comment]
Yeah... if no target is specified all ES2015+ code will be transformed (same as preset-latest behavior)?

**Comment by dilyanpalauzov** [comment]
There is source language and target language. 

“will transform all ECMAScript 2015+ code by default.” means that the source language is ES2020, e.g. the input can containing optional chaining.  I  mean, I read it as “transform FROM". It does not say anything about ”transform TO (version)”.

In its current writing “transform” is transitive verb, https://www.merriam-webster.com/dictionary/transform .  But for the “to become transformed” the intransitive verb is needed.

E.g. "preset-env transforms A” means “preset-evn changes A”.  It does not say to what is A changed.

**Comment by existentialism** [comment]
Happy to change the language to reduce confusion, would you like to send a PR?

**Comment by dilyanpalauzov** [comment]
My reading is that targets: accepts only browserslist queries and the browserslist does not accept "es2020" as input.  So targets can be only a list of browsers with corresponding versions. `ES2015+` is not a browserslist query and I cannot write what the default for `targets` is.

So something like this:
> Sidenote, if no targets are specified, the default is `targets:{a browserslist query}`.
or
> Sidenote, if no targets are specified, the default is not to transpile.
or
> Sidenote, if no targets are specified, the default is only to transpile code that has not reached stage 4 in TC39.
